TouchEffect/Interfaces/ITouchEff.cs

```diff
@@ -65,6 +65,18 @@ public interface ITouchEff
 
         bool IsCompletedSet { get; }
 
+        bool NativeAnimation { get; }
+
+        Color NativeAnimationColor { get; }
+        
+        bool AndroidRipple { get; }
```

Comment:
Hm, what is the purpose of having both properties (AndroidRipple and NativeAnimation) ?

Reply:
Well NativeAnimation enables native animations on every platform, but `AndroidRipple` and `UWPTilt` make it possible to disable the native animations on specific platforms. It gives more control if ever you want to, for example, enable native animations on every platform, but have a custom animation on a particular platform.

Reply:
So, if I want to have android native animation, I need to enable both properties anyway.
The same for UWP.. so, I do not see any reason to leave both.

Have I understood it correctly?

Reply:
No, enabling just `NativeAnimation` is enough to enable animations on every single platform where it's available. However if you want, you can then disable them on a platform-specific basis with AndroidRipple and UWPTilt (because they are both true by default)

Reply:
Got it
But then remove AndroidRipple and UWPTilt.

Because I can not enable them, instead of Enable for all and disable specific.